packages/commonwealth/client/scripts/views/components/sidebar/CollapsableSidebarButtonUnscoped.tsx

```diff
@@ -0,0 +1,33 @@
+import 'components/sidebar/collapsable_button.scss';
+import React, { useState } from 'react';
+import useSidebarStore from '../../../state/ui/sidebar/index';
+import { CWIconButton } from '../component_kit/cw_icon_button';
+
+export const CollapsableSidebarButtonUnscoped = () => {
+  const { setMenu, menuName, menuVisible, setUserToggledVisibility } =
+    useSidebarStore();
+
+  const [isCollapsed, setIsCollapsed] = useState(false);
```

Comment:
why do you need additional state if we already have global state to check if sidebar is visible or not. use `menuVisible` instead and get rid of this state

Reply:
The issue is that menuVisible seems not to control the sidebar visibility. As the sidebar doesn't react to it but instead always starts off as uncollapsed, and changes to collapsed.